### PR TITLE
generic train model and feature editing from .json

### PR DESCRIPTION
**Pull request by caglanakpinar**
you can add a new model of .py file with its class and prediction and train model. All you need to add a new model into the learning_models.json file with a new Obj.
model_processor.py is added which handles the prediction and training process of models at learning_models.json file
you can assign the model of output column at the test set from learning_models.json and by using pred_fields and by using uid you may show with which columns you want to merge into existed data set.

### CODE REVIEW

File: configs.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import datetime
 
 
-
 is_local_run = False
 is_min_max_norm = True
 run_from_sample_data = False
-sample_args = ['main.py', 'feature_engineering_lstm', 'all']
+run_gpu = False
+sample_args = ['main.py', 'train_process', '0', 'all']
```
Comment (caglanakpinar) [inline configs.py:8]: This is for when you are running a GPU server but you are not actually using the GPU for Tensorflow. I had an error about saving the Tensorflow model after using multi GPU cores. that is why added in order to make it optional

File: configs.py [hunk 2]
```diff
@@ -16,6 +16,8 @@
 hyper_parameter_path = 'hyper_parameters.json'
 feature_lstm_ae = 'features_lstm_ae.json'
 feature_path = 'features.json'
+learning_model_path = 'learning_models.json'
+pred_data_path = 'predicted_test_data.csv'
```
Comment (caglanakpinar) [inline configs.py:20]: This are for a new generic style. You need to show where you put your file which has information about the model you may want to predict or train. 'pred_data_path' is the path you can save your .csv file with the prediction. make sure you run "main.py prediction 0/1 all" in order to get all models you have implemented with the argument is 'all'. Otherwise, you only get the prediction of a specific model.

File: data_access.py [hunk 3]
```diff
@@ -73,4 +75,12 @@ def decide_feature_name(feature_path):
...
\ No newline at end of file
+    return feature_2
+
+
+def callfunc(my_file):
+    pathname, filename = os.path.split(my_file)
+    sys.path.append(os.path.abspath(pathname))
+    modname = os.path.splitext(filename)[0]
+    my_mod = importlib.import_module(modname)
+    return my_mod
```
Comment (caglanakpinar) [inline data_access.py:86]: This collects the .py file for the model you show the framework on learning_models.json
this function finds the model of .py file its functions and classes to run on model_processor.py
